iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/AttachLoad.java

```diff
@@ -0,0 +1,265 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.iidm.modification.topology;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.reporter.Report;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.reporter.TypedValue;
+import com.powsybl.computation.ComputationManager;
+import com.powsybl.iidm.modification.NetworkModification;
+import com.powsybl.iidm.network.*;
+import com.powsybl.iidm.network.extensions.BusbarSectionPosition;
+import com.powsybl.iidm.network.extensions.ConnectablePosition;
+import com.powsybl.iidm.network.extensions.ConnectablePositionAdder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+import java.util.stream.Stream;
+
+import static com.powsybl.iidm.modification.ModificationUtils.throwExceptionOrLogError;
+import static com.powsybl.iidm.modification.topology.TopologyModificationUtils.*;
+
+/**
+ * This method adds a new load on an existing voltage level. The voltage level should be described
+ * in node/breaker topology.
+ * @author Coline Piloquet <coline.piloquet at rte-france.com>
+ */
+public class AttachLoad implements NetworkModification {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AttachLoad.class);
+
+    private final LoadAdder loadAdder;
+    private final String voltageLevelId;
+    private String bbsId;
+    private int loadPositionOrder;
+    private PositionInsideSection loadPositionInsideSection = PositionInsideSection.SPECIFIC;
```

Comment:
what do you mean?

Reply:
I mean that `AttachLoad` should have only an explicit `loadPositionOrder` I think, and not the current pair `loadPositionInsideSection`, `loadPositionOrder`. The enum `PositionInsideSection` is there to help the user which does not know the orders in his voltage level, I think it would make more sense to put that apart that class.

Reply:
I moved this into a method in TopologyModificationUtil. (Not sure if it’s the right place though). Now the loadOrderPosition must be provided.